model/src/voice/close_code.rs

```diff
@@ -0,0 +1,143 @@
+use serde_repr::{Deserialize_repr, Serialize_repr};
+use std::convert::TryFrom;
+use std::{
+    error::Error,
+    fmt::{Display, Formatter, Result as FmtResult},
+};
+
+/// Voice gateway close event codes.
+#[derive(
+    Clone, Copy, Debug, Deserialize_repr, Eq, Hash, PartialEq, PartialOrd, Ord, Serialize_repr,
+)]
+#[non_exhaustive]
+#[repr(u16)]
+pub enum CloseCode {
+    /// An invalid opcode was sent.
+    UnknownOpcode = 4001,
+    /// A payload was sent prior to identifying.
```

Comment:
Can you add 4002 here as well as it was just documented 
https://github.com/discord/discord-api-docs/pull/1818